app/lib/query_count_job_logger.rb

```diff
@@ -2,13 +2,30 @@
 require 'sidekiq/job_logger'
 
 class QueryCountJobLogger < ::Sidekiq::JobLogger
```

Comment:
Since this now does more than just add query counts I reckon we can do the following

```suggestion
class CustomLogger < ::Sidekiq::JobLogger
```

Reply:
Done 👍